Merging

Sl3v7n opened the pull request:
@carledriss Hello Carlos I've made the marge for the project when you had a time could take a look at it, I would like to know if the merge works fine.

Project including create edit and delete of a new project

Commit:
Test

Commit:
Test

Commit:
Project tests and modifications it has create project, edit and delete the project

Commit:
Project tests and modifications it has create project, edit and delete the project

sperBruno committed:
Story tests

* update story

* update

* update

* update

* update test

* romoving credential

* update Carlos' observation

* Update Test

Commit:
Project tests and modifications it has create project, edit and delete the project

Commit:
Project tests and modifications it has create project, edit and delete the project

carledriss commented:
@Sl3v7n 
Please update against upstream/develop

Sl3v7n committed:
Merge branch 'develop' into develop

carledriss commented on gradle.properties:
```diff
@@ -1,7 +1,12 @@
+projectName=
+accountName=
+newprojectName=
+newdescriptionName=
```
In fact you don't need to put the projectName, accountName in the properties file
Only put in this file common variables in the environment.

you can declare them directly in the tests

carledriss replied:
Please fix this observations

carledriss commented on gradle.properties:
```diff
@@ -1,7 +1,12 @@
 #Settings to PivotalTracker - UI
-authenticationBaseURL=https://www.pivotaltracker.com
+
+authentication.baseURL=https://www.pivotaltracker.com
```
this should be authenticationBaseURL
why did you change it?

Sl3v7n replied:
LOL, I wasn't aware that I've changed that line,... Probably I change it by mistake

carledriss commented on src/test/java/project/ProjectTests.java:
```diff
@@ -0,0 +1,66 @@
+        projectManagement.setNewProjectDescription(newDescription);
+        projectManagement.clickSave();
+        //then
+        Assert.assertEquals(newProjectName, projectManagement.getNewProjectName());
```
you can use a static import for Asserts

Sl3v7n replied:
String ProjectName = "";
Assert.assertEquals(newProjectName, ProjectName);

something like this

carledriss replied:
no
```java
import static org.junit.Assert.assertEquals;
```

and in the test
```java
assertEquals(newProjectName, ProjectName);
```

Sl3v7n committed:
Update gradle.properties

fixing authenticationBaseURL

carledriss commented:
@Sl3v7n 
You have a lot of checkstyle issues.
Please fix them.

Commit:
Gradle checked

Commit:
Gradle checked

carledriss commented:
after fixing minor issues.
LGTM!
anyone of you @SeleniumCourse/team1  can squash and merge this Pull Request

Commit:
Gradle checked

Commit:
Gradle checked

Sl3v7n committed:
Merge branch 'develop' into develop

carledriss commented on src/main/java/org/fundacionjala/pivotal/utils/Environment.java:
```diff
@@ -243,4 +248,41 @@ public String getRemotePlatformVersion() {
+     */
+    public String getDescriptionName() {
+        return getEnv(DESCRIPTION_NAME);
+    }
```
Do we still need those methods?

carledriss commented on src/test/java/project/ProjectTests.java:
```diff
@@ -0,0 +1,81 @@
+    static final String PROJEC_NAME = "";
+    static final String ACCOUNT_NAME = "";
+    static final String NEW_PROJECT_NAME = "";
+    static final String NEW_DESCRIPTION_NAME = "";
```
Set random values for these variables

carledriss commented:
@Sl3v7n 
you still have checkstyle issues

Commit:
test

# Conflicts:
#	src/main/java/org/fundacionjala/pivotal/ui/pages/common/CommonActions.java

Commit:
test

# Conflicts:
#	src/main/java/org/fundacionjala/pivotal/ui/pages/common/CommonActions.java

Commit:
Merge remote-tracking branch 'origin/develop' into develop1

# Conflicts:
#	src/main/java/org/fundacionjala/pivotal/ui/pages/common/CommonActions.java

Commit:
Gradle checked

Commit:
Merge remote-tracking branch 'origin/develop' into develop1

# Conflicts:
#	src/main/java/org/fundacionjala/pivotal/ui/pages/common/CommonActions.java
#	src/main/java/org/fundacionjala/pivotal/ui/pages/project/ProjectForm.java
#	src/main/java/org/fundacionjala/pivotal/ui/pages/project/ProjectManagement.java
#	src/main/java/org/fundacionjala/pivotal/utils/Environment.java
#	src/test/java/project/ProjectTests.java

Commit:
Gradle checked

Sl3v7n commented:
@carledriss the push was successful finally  sorry for the delay

carledriss commented:
LGTM! :+1:
